servers/src/main/java/tachyon/worker/netty/DataServerHandler.java

```diff
@@ -109,6 +110,13 @@ private void handleBlockReadRequest(final ChannelHandlerContext ctx,
       ChannelFuture future = ctx.writeAndFlush(resp);
       future.addListener(ChannelFutureListener.CLOSE);
       return;
+    } catch (TimeoutException tee) {
+      LOG.error("Failed to lock block: " + blockId, tee);
```

Comment:
Since we have split out the error cases, we can probably make the log messages more informative? (Line 107 and 114)